qutebrowser/browser/commands.py

```diff
@@ -1647,6 +1650,11 @@ def on_editing_finished(self, elem, text):
         except webelem.Error as e:
             raise cmdexc.CommandError(str(e))
 
+    def on_editor_orphaned(self, ed):
```

Comment:
Hmm, while the reconnecting works, I wonder whether there isn't a better solution. Can't we detect whether the view is gone in `on_editing_finished` instead?

Reply:
The C++ object wrapped by the `WebEngineView` is deleted. A few ideas I had are:

- Can a `WebEngineView` check if the C++ object is deleted?
- Does `AbstractTab` expose any way to check if it is closed?
- Can I search the `TabbedBrowser` to see if it still contains a tab I hold a reference to?

I didn't see an obvious way to do any of these, so catching the shutdown signal from the tab seemed like the best solution. Any ideas?

Reply:
You can use `sip.isdeleted` on a PyQt object. So I guess adding a `is_deleted` method to `AbstractTab` which does `return sip.isdeleted(self._widget)` or so would be the nicest solution here - what do you think?